centreon/packages/ui-context/src/types.ts

```diff
@@ -5,8 +5,22 @@ export enum ListingVariant {
   extended = 'extended'
 }
 
+export enum DashboardGlobalRole {
+  administrator = 'administrator',
+  creator = 'creator',
+  viewer = 'viewer'
+}
+
+export interface DashboardRolesAndPermissions {
+  administrateRole: boolean;
+  createRole: boolean;
+  globalUserRole: DashboardGlobalRole;
+  viewRole: boolean;
+}
+
```

Comment:
```suggestion
export interface DashboardRolesAndPermissions {
  manageAllDashboards: boolean;
  createDashboards: boolean;
  globalUserRole: DashboardGlobalRole;
  viewDashboards: boolean;
}
```

permissions are mixed with roles (cfr. Slack discussion)

→ `viewer` **role** has the `viewDashboards` **permission**